src/pages/net-neutrality/petition.js

```diff
@@ -9,26 +12,48 @@ import SignupForm from '../../components/net-neutrality/signup-form.js';
 import FormBody from '../../components/signup-form/form-body.js';
 import Modal from '../../components/modal.js';
 
+const DONATE_CTA_DELAY = 750; // in milliseconds
+
 var Signup = React.createClass({
   getInitialState: function() {
     return {
       showModal: false,
+      dismissedModal: false,
       signupSuccess: false
     };
   },
   componentDidMount: function() {
-    /*if (!this.props.location.query.subscribed) {
-      setTimeout(() => {
-        this.setState({
-          showModal: true
-        });
-      }, 3000);
-    }*/
+    if (typeof window !== "undefined" && window.addEventListener) {
+      this._withScroll = e => this.handleScroll(e);
+      window.addEventListener('scroll', this._withScroll);
+    }
+  },
+  componentWillUnmount() {
+    if (this._withScroll) {
+      window.removeEventListener('scroll', this._withScroll);
+    }
+  },
+  handleScroll: function(e) {
+    if (this._ctaTimeout) {
+      clearTimeout(this._ctaTimeout);
+    }
+    this._ctaTimeout = setTimeout(() => this.spawnDonateCTA(), DONATE_CTA_DELAY);
+  },
+  spawnDonateCTA: function() {
+    if (!storage.getItem('dismissedModal')) {
+      this.setState({
+        showModal: true
+      });
+    }
+  },
+  donateClicked: function() {
```

Comment:
let's remove this, it's not something we need to manually track since we GA automatically tracks this on the donate side.